index.html

```diff
@@ -525,5 +525,52 @@ <h2>Privacy Considerations</h2>
     <section>
       <h2>Security Considerations</h2>
     </section>
+
+    <section class="appendix informative">
+<h2>IANA Considerations</h2>
+<section id="media-type">
+  <h2>The <code>application/credential-1.1+json</code> Media Type</h2>
```

Comment:
```suggestion
  <h2>The <code>application/credential+json-1.1</code> Media Type</h2>
```
might be an option

Reply:
This would be saying that we are using `json-1.1`, does not make sense.

Could do this:

```html
<code>application/credential+json; version=1.1</code>
```

or 

```html
<code>application/credential+json; profile=https://www.w3.org/TR/2022/REC-vc-data-model-20220303/</code>
```

Reply:
@mprorock I am resolving this, but if you disagree, just reopen.